mods/cnc/maps/nod06a/nod06a.lua

```diff
@@ -0,0 +1,235 @@
+Gdi5Waypoints = { waypoint1, waypoint4 }
+Gdi6Waypoints = { waypoint2, waypoints3 }
+
+Grd1TriggerFunctionTime = DateTime.Seconds(3)
+
+Grd1TriggerFunction = function()
+	MyActors = Utils.Take(2, GDI.GetActorsByType('mtnk'))
+	Utils.Do(MyActors, function(actor)
+		MovementAndHunt(actor, Gdi3Waypoints)
+	end)
+end
+
+Grd2TriggerFunction = function()
+	if not Grd2Switch then
+		for type, count in pairs({ ['e1'] = 2, ['e2'] = 1, ['jeep'] = 1 }) do
+			MyActors = Utils.Take(count, GDI.GetActorsByType(type))
+			Utils.Do(MyActors, function(actor)
+				MovementAndHunt(actor, Gdi4Waypoints)
+			end)
+		end
+		Grd2Swicth = true
+	end
+end
+
+Atk1TriggerFunction = function()
+	if not Atk1Switch then
+		for type, count in pairs({ ['e1'] = 3, ['e2'] = 3, ['jeep'] = 1 }) do
+			MyActors = Utils.Take(count, GDI.GetActorsByType(type))
+			Utils.Do(MyActors, function(actor)
+				MovementAndHunt(actor, Gdi5Waypoints)
+			end)
+		end
+		Atk1Switch = true
+	end
+end
+
+Atk2TriggerFunction = function()
+	if not Atk2Switch then
+		for type, count in pairs({ ['mtnk'] = 1, ['jeep'] = 1 }) do
+			MyActors = Utils.Take(count, GDI.GetActorsByType(type))
+			Utils.Do(MyActors, function(actor)
+				MovementAndHunt(actor, Gdi6Waypoints)
+			end)
+		end
+		Atk2Switch = true
+	end
+end
+
+Atk3TriggerFunction = function()
+	if not Atk3Switch then
+		local list = Nod.GetGroundAttackers()
+		AirstrikeProxy = Actor.Create("airstrike.proxy", true, { Owner = GDI })
+		AirstrikeProxy.SendAirstrike(list[DateTime.GameTime % #list + 1].CenterPosition, false, 256 - 28)
+		AirstrikeProxy.Destroy()
+		Atk3Switch = true
+	end
+end
+
+Chn1TriggerFunction = function()
+	Reinforcements.ReinforceWithTransport(GDI, 'tran', Chn1Units, Chn1Waypoints, nil,
+	function(transport, cargo)
+		transport.UnloadPassengers()
+		Utils.Do(cargo, function(actor)
+			IdleHunt(actor)
+		end)
+	end,
+	function(transport)
+		transport.Move(waypoint14.Location)
+		transport.Destroy()
+	end)
+end
+
+Chn2TriggerFunction = function()
+	Reinforcements.ReinforceWithTransport(GDI, 'tran', Chn2Units, Chn2Waypoints, nil,
+	function(transport, cargo)
+		transport.UnloadPassengers()
+		Utils.Do(cargo, function(actor)
+			IdleHunt(actor)
+		end)
+	end,
+	function(transport)
+		transport.Move(waypoint14.Location)
+		transport.Destroy()
+	end)
+end
+
+Obj2TriggerFunction = function()
+	Nod.MarkCompletedObjective(NodObjective2)
+	Reinforcements.Reinforce(Nod, Obj2Units, { Obj2UnitsEntry.Location, waypoint13.Location }, 15)
+end
+
+MovementAndHunt = function(unit, waypoints)
+	if unit ~= nil then
+		Utils.Do(waypoints, function(waypoint)
+			unit.AttackMove(waypoint.Location)
+		end)
+		IdleHunt(unit)
+	end
+end
+
+SendHarvester = function()
+	local Unit = Map.ActorsInBox(Map.TopLeft, Map.BottomRight, function(actor)
+		return actor.Owner == GDI and actor.Type == 'harv'
+	end)
+	if #Unit > 0 then
+		Unit[1].Move(waypoint8.Location)
+		Unit[1].FindResources()
+	end
+end
+
+InserNodUnits = function()
```

Comment:
Typo? `InserNodUnits` -> `InsertNodUnits`

You should also place `Camera.Position = UnitsRallyRight.CenterPosition` in here.